src/zzz_od/config/game_config.py

```
@@ -262,7 +237,7 @@ def get_action_keys(self, control_method: str) -> dict[str, str]:
         """获取指定控制方式的所有按键映射。
 
         Args:
-            control_method: ControlMethodEnum 的值，如 'keyboard' / 'xbox' / 'ds4'。
+            control_method: 控制器类型，如 'keyboard' / 'xbox' / 'ds4'。
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)